portal/templates/dashboard/deals.njk

```diff
@@ -113,53 +81,102 @@
-        text: "Exporter",
-        attributes: {
-        "data-cy": "deal__header--exporter",
-        "aria-sort": "none"
-      }
-      },
-      {
-        text: "Bank ref",
-        attributes: {
-        "data-cy": "deal__header--bankRef"
-      }
-      },
-      {
-        text: "Product",
-        attributes: {
-        "data-cy": "deal__header--product"
-      }
-      },
-      {
-        text: "Type",
-        attributes: {
-        "data-cy": "deal__header--type"
-      }
-      },
-      {
-        text: "Status",
-        attributes: {
-        "data-cy": "deal__header--status"
-      }
-      },
-      {
-        text: "Last updated",
-        attributes: {
-        "data-cy": "deal__header--updated"
-      }
-      }
-    ],
-    rows: items
-  }) }}
+  <div class="moj-filter-layout">
+
+    {{dashboardFiltersPanel.render({
+      filters: filters,
+      selectedFilters: selectedFilters,
+      keyword: keyword
+    })}}
+
+    <div class="moj-filter-layout__content">
+
+      {{dashboardFitlersActionBar.render({
+        createdByYou: createdByYou,
+        selectedFilters: selectedFilters
+      })}}
+
+      <div class="moj-scrollable-pane">
+
+        <div class="moj-scrollable-pane__wrapper">
+
+          {{ govukTable({
+            classes: "govuk-table govuk-!-margin-bottom-0",
+            attributes: {
+            'data-module': 'moj-sortable-table'
+          },
+            head: [
+              {
+                text: "Exporter",
+                attributes: {
+                "data-cy": "deal__header--exporter",
+                "aria-sort": "none"
+              }
+              },
+              {
+                text: "Bank ref",
+                attributes: {
+                "data-cy": "deal__header--bankRef"
+              }
+              },
+              {
+                text: "Product",
+                attributes: {
+                "data-cy": "deal__header--product"
+              }
+              },
+              {
+                text: "Type",
+                attributes: {
+                "data-cy": "deal__header--type"
+              }
+              },
+              {
+                text: "Status",
+                attributes: {
+                "data-cy": "deal__header--status"
+              }
+              },
+              {
+                text: "Last updated",
+                attributes: {
+                "data-cy": "deal__header--updated"
+              }
+              }
+            ],
+            rows: items
+          }) }}
+
+          {{ pagination.pagination(totalPages = pages.totalPages, currentPage = pages.currentPage, totalItems = pages.totalItems, paginationRoot = '/dashboard/deals' ) }}
+
+        </div>
+
+      </div>
+    </div>
+  </div>
+
+{% endblock %}
 
-  {{ pagination.pagination(totalPages = pages.totalPages, currentPage = pages.currentPage, totalItems = pages.totalItems, paginationRoot = '/dashboard/deals' ) }}
 
+{% block scripts %}
+  <script>
```

Comment:
can we add this as an external script instead? so we load it like `<script src="path-to-script-in-public-folder"></script>`

Reply:
Done :) Had to add it to index.njk, couldn't get it to work in deals.njk (any ideas?)